gulpfile.js

```diff
@@ -141,7 +143,21 @@ function lintTask() {
     rules: {
       'complexity': [1, 6],
       'max-statements': [1, 30]
-    }
+    },
+    globals: [
```

Comment:
I doubt it, there were no previous issues with those globals so it would only affect future changes. An alternative would be to run a lint on the source and tests separately.